tests/vmi_multus_test.go

```diff
@@ -838,9 +843,8 @@ var _ = Describe("[Serial]SRIOV", func() {
 			// start peer machines with sriov interfaces from the same resource pool
 			// manually configure IP/link on sriov interfaces because there is
 			// no DHCP server to serve the address to the guest
-			const networkName = "sriov-link-enabled"
-			vmi1 := getSriovVmi([]string{networkName}, cloudInitNetworkDataWithStaticIPsByMac(networkName, mac1.String(), cidrA))
-			vmi2 := getSriovVmi([]string{networkName}, cloudInitNetworkDataWithStaticIPsByMac(networkName, mac2.String(), cidrB))
+			vmi1 := getSriovVmi([]string{sriovnet3}, cloudInitNetworkDataWithStaticIPsByMac(sriovnet3, mac1.String(), cidrA))
```

Comment:
why not use here `sriovnet` ?
you have just one network of SRIOV

Reply:
Look [here](https://github.com/kubevirt/kubevirt/pull/4620/commits/29d36d71d8b3009a97cd618554207f136cc31186#diff-ac233e621be1a1855fe741915d860d91484c7f061fc42beb213f4f9e0a756f86R628).

There are 3 networks, not one. The 3rd is even differently defined.

Reply:
Right, so does it worth to add a comment where you declare `sriovnet3` that it will use sriovLinkEnableConfCRD 
or that it is `link-enabled` ?

Reply:
I think it is worth localizing the definition, creation and the whole SRIOV stuff better.
I do not want to open this PR to that big change.